Strategies/ITVision.md

```diff
@@ -0,0 +1,134 @@
+# ESDC's IT Vision
+
+[HDLC](HumanDevelopmentLifeCycle.md) -&gt; [KPIs](IITB_KPIs.md) -&gt; IT Vision -&gt; Guiding Principles -&gt; IT Strategies (roadmap)
+
+The following is ESDC's IT Vision.
+It paints a vision of the future of IT within ESDC.
+It is **not** meant to be time bound, but rather as an ideal state to which ESDC IT needs to strive to get to.
+Such state will continuously evolve when new information and knowledge is captured, re-adjusting the vision in the process.
+
+The IT Vision complements the [HDLC](HumanDevelopmentLifeCycle.md) story which is taken from the point of view of the citizen, and the [KPIs](IITB_KPIs.md) which define the performance metrics needed to sustain the HDLC, by providing the point of view of the organization's IT capabilities.
+
+The IT Vision leverages the work done in the [IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx), ensures alignment with [IITB's high level priorities](http://esdc.prv/en/iitb/corporate/Who_We_Are/Plans_and_Priorities/index.shtml), and brings the [GC's Strategic Plan for IM/IT 2017-2021](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html) into ESDC's operational context.
+
+The ESDC IT Vision is explained using the [GC's Strategic Plan for IM/IT 2017-2021](https://www.canada.ca/en/treasury-board-secretariat/services/information-technology/strategic-plan-2017-2021.html#toc4) six vision pillars:
```

Comment:
The GC Strategic Plan sort IM/IT 2017 was updated in the [Digital Operations Strategic Plan 2018](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html).  We probably should reference that one

Reply:
@smellems 
SHHHIIIIIIIIII..... t'as raison! 
Leur nouveau plan stratégique est organisé très différemment. Je vais faire beaucoup de modifications d'éditions dans un nouveau commit. Le contenu sera pareil mais structuré pour s'aligné avec le GC Digital Ops Plan (i.e. même 6 chapitres)

Reply:
see commit #cab6d14